src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -85,6 +87,9 @@ export default function Autocomplete({
 
   if (isEqual) {
     optionalArgs.isOptionEqualToValue = isEqual;
+  } else {
+    optionalArgs.isOptionEqualToValue = (option: unknown, value: unknown) =>
```

Comment:
This ensures that if clients pass in an empty string as the selected value, which creates a warning, that it is still considered equal to `undefined`.